app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = t("static_pages.full_title")
+    if page_title.blank?
```

Comment:
dùng toán tử 3 ngôi cho đoạn if ngắn e nhé
```suggestion
   page_title.blank? ? base_title : "#{page_title} | #{base_title}"
```